pkg/controller/installation/installation_controller.go

```diff
@@ -363,6 +391,12 @@ func (r *ReconcileInstallation) preflightChecks(installation *integreatlyv1alpha
 
 	result := reconcile.Result{}
 	requiredSecrets := []string{"github-oauth-secret"}
```

Comment:
Not sure how the `github-oauth-secret` gets created. If we've got the ability, it might be good to add the `owner: integreatly` label to it so that it gets watched like the SMTP secret.

Reply:
We don't need these labels now -- I've changed it to watch all secrets in the namespace.